chart/githubconnector/values.yaml

```diff
@@ -1,11 +1,13 @@
 replicaCount: 1
 
 container:
-  image: pprecel/github-connector:0.0.1
+  image: karoljaksik/github-connector:1.0.2
   containerPort: 8080
   limits:
     memory: "128Mi"
     cpu: "500m"
 
 service:
   port: 80
+
+kymaAddress: 35.195.198.66.xip.io
```

Comment:
I think that in future this should be imported from environment.

Reply:
It's in team's backlog as a task for future